modules/ROOT/pages/configure-client-management-openid-task.adoc

```diff
@@ -47,7 +47,39 @@ The *External Identity* page appears.
  In *Advanced Settings* you can also select:
   *** *Disable server certificate validation*: Disables server certificate validation if your OpenID client management instance presents a self-signed certificate, or one signed by an internal certificate authority.
   *** *Enable client deletion in Anypoint Platform*: Enables deletion of clients created with this integration.
-  *** *Enable client deletion and updates in IdP*: To use this option, you must also select the *Enable client deletion in Anypoint Platform* option. This option enables you to update and delete external clients in the configured IdP through an outbound call made by Anypoint Platform. 
+  *** *Enable client deletion and updates in IdP*: To use this option, you must also select the *Enable client deletion in Anypoint Platform* option. This option enables you to update and delete external clients in the configured IdP through an outbound call made by Anypoint Platform to `{clientRegistrationUrl}/{clientID}`, for example: `+https://example.salesforce.com/services/oauth2/6779ef20e75817b79602+`. 
+   **** The defined `Client Registration URL` is the UI through which both `PUT` and `DELETE` requests are made.  
+   **** The `Authorization:` header is included as part of the request only if the `Authorization Header` under *Advanced settings* is configured.
+   **** The `client_id` request parameter that is passed is the same as the `client_id` passed in the `PUT` request payload.
+   **** The `token_endpoint_auth_method` passed in the payload is always `client_secret_basic`.
+   **** Only the `Client Registration URL` and `Authorization Header` come from what is configured in OIDC *Client management*. +
+This is an example of the payload for a `PUT` (update) request: 
++
+----
+PUT /oauth2/connect/register/{{client_id}}
+Accept: application/json
+Host: https://identity.example.com
+Authorization: Bearer access-token
+ {
+     "client_id": "client_id",
+     "client_secret": "some-secret",
+     "redirect_uris": [
+          "test-redirect-uri"
+      ],
+     "grant_types": [ ],
+     "token_endpoint_auth_method": "client_secret_basic",
+     "response_types": [ ],
+     "client_name": "test-client-name"
```

Comment:
I think this is fairly obvious, but this field doesn't map cleanly to the field that you would be PATCHing on the inbound side: it's `name` there. I think we could add a note here.